components/layout.module.css

```diff
@@ -49,6 +49,10 @@
     overflow: visible !important;
 }
 
+.profileContainer > img {
+    overflow: hidden !important;
```

Comment:
I'm not sure what happened here, but the main profile picture on the home page was showing as square instead of round. This wasn't directly associated with the issue, but it was a nice to have.